content/vendor/packaging/promoting-releases.md

```diff
@@ -35,11 +35,18 @@ Additionally, each promoted release must be given a required status (required or
 
 Semantic versioning is available in KOTS v1.58.0 and later.
 
-You can use the semantic version format for the version label that you assign to a release in the admin console. For more information about semantic versioning, see [Semantic Versioning 2.0.0](https://semver.org). 
-In order for this label to be used to sequence releases on a channel, semantic versioning must be enabled by turning on the `Require semver for releases promoted to this channel` toggle in the channel settings.
-If you use semantic versioning on a channel, we recommend that you always use semantic versioning on that channel.
-This toggle is disabled by default.
+### Enabling Semantic Versioning
+
+Semantic versioning must be enabled on a per-channel basis. It can be enabled for existing channels by enabling **Enable Semantic Versioning** in the channel settings in the vendor portal. When creating a new channel in the vendor portal, select **Prevent promoting releases with an invalid semantic version to this channel** to enable this option. Semantic versioning can also be enabled and disabled for an existing channel with the [`kots channel enable-semantic-versioning`](/vendor/cli/channel-enable-semantic-versioning) and [`kots channel disable-semantic-versioning`](/vendor/cli/channel-disable-semantic-versioning) commands, respectively.
```

Comment:
Do you think the second sentence here has the word enabled/enabling too many times in one sentence now?

Reply:
Argh, good call. Need more coffee. I'll do another edit.